app/src/services/landmarkExtractor.ts

```
@@ -0,0 +1,35 @@
+  const tmpDir = FileSystem.cacheDirectory + 'frames_' + Date.now() + '/';
+  await FileSystem.makeDirectoryAsync(tmpDir, { intermediates: true });
+  await FFmpegKit.execute(`-i ${videoPath} ${tmpDir}frame_%04d.png`);
+  const files = await FileSystem.readDirectoryAsync(tmpDir);
+  const results: number[][][] = [];
+  for (const f of files) {
+    try {
+      const data = await FileSystem.readAsStringAsync(tmpDir + f, {
+        encoding: FileSystem.EncodingType.Base64,
+      });
+      const out = handModel.runSync([{ uri: `data:image/png;base64,${data}` }]) as any[];
+      if (out && out[0]) results.push(out[0] as number[][]);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)